.circleci/config.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       - run:
           name: PHP 5.5 Magento 1.8
           command: |
-            PHPUNIT_PHAR=tests/unit/phpunit-4.8.36.phar MAGENTO_VERSION=magento-mirror-1.8.1.0 tests/scripts/ci.sh
+            PHPUNIT_PHAR=tests/integration/phpunit-4.8.36.phar MAGENTO_VERSION=magento-mirror-1.8.1.0 tests/scripts/ci.sh
```

Comment:
I think having "integration" test in unit-php55-mage18 build is a bit weird. We have true integration test running webdriver on http://magento1-sandbox.bolt.me so I feel like every test in this directory is "unit" test in a sense it doesn't integrate with real Bolt APIs.